x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/mapper/SemanticTextFieldMapper.java

```diff
@@ -383,6 +384,11 @@ public IndexFieldData.Builder fielddataBuilder(FieldDataContext fieldDataContext
             throw new IllegalArgumentException("[semantic_text] fields do not support sorting, scripting or aggregating");
         }
 
+        @Override
+        public final boolean fieldHasValue(FieldInfos fieldInfos) {
```

Comment:
Any particular reason why you chose to make this `final`?

Reply:
Good catch, it won't be required. For some reason I got an error message that it can not be overridden, that's why I put the final keyword.